packages/4337-local-bundler/package.json

```diff
@@ -9,14 +9,17 @@
     "4337-local-bundler-test": "dist/bin/test.js"
   },
   "scripts": {
-    "build": "npx rimraf dist && tsc && chmod +x dist/bin/*.js",
+    "clean": "npx rimraf dist",
+    "build": "tsc && chmod +x dist/bin/*.js",
     "lint": "eslint .",
-    "prepack": "npm run build",
-    "prepublish": "npm run build",
-    "postinstall": "npm run build"
+    "preprepare": "npm run clean",
+    "prepare": "npm run build"
```

Comment:
Prepare gets run at all the same lifetimes as these scripts. Cleaning and building multiple times within an `npm ci` was causing issues with the build process, where small changes to the dependency graph could make the order in which the build happens change and cause errors.